frontend/dist/clouwaypush.js

```diff
@@ -156,21 +119,38 @@ angular.module('clouway-push', [])
 
       var subscribeForEvent = function (eventName, correlationId) {
         var subscriber = service.openConnection();
-        connectionMethods.bind(subscriber, eventName, correlationId);
+        var params = {
+          subscriber: subscriber,
+          eventName: eventName,
+          correlationId: correlationId
+        };
+
+        $http.put(endpoints.serviceUrl, '', {params: params}).then(function (data) {
```

Comment:
Here should be a check if the serviceUrl is empty(or not defined) to not send a request.